content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -281,6 +281,52 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+## IPv4/IPv6 dual stack
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Alpha features change rapidly. {{< /warning >}}
+
+IPv4/IPv6 dual stack enabled Kubernetes clusters allow the assignment of both IPv4 and IPv6 addresses simulatenously which enables the following capabilities:
```

Comment:
there is a bit of phrasing issue with `enable kubernetes cluster allow`.

Reply:
I re-read this and the complete sentence seems okay. Could you provide an alternative that makes it more clear?

Reply:
I reckon this fine to leave as-is, then.